pkg/virt-handler/cgroup/cgroup.go

```diff
@@ -141,6 +144,23 @@ func getCpuSetPath(manager Manager, cpusetFile string) (string, error) {
 	return cpusetStr, nil
 }
 
+// set cpus "cpusList" on the allowed CPUs. Optionally on a subcgroup of
+// the pods control group (if subcgroup != nil).
+func SetCpuSet(manager Manager, subcgroup string, cpusList []int) error {
```

Comment:
This was done because the code is the same, therefore can be shared. With that in mind,
creating two private functions would just duplicate code.

But perhaps you are have a point, unsure.

Reply:
Not sure if I understand. This function is being used only once in `pkg/virt-handler/vm.go`:
```go
err = cgroup.SetCpuSet(cgroupManager, "housekeeping", []int{hkcpu})
if err != nil {
	return err
}
```

So, instead it could be used as:
```go
err = cgroupManager.SetCpuSet("housekeeping", []int{hkcpu})
if err != nil {
	return err
}
```

The same goes for `GetCgroupThreads`. But in any case I can do it for you in a follow-up PR, so feel free to ignore this.

Reply:
ping